lib/logstash/filters/kv.rb

```diff
@@ -335,6 +361,26 @@ def has_value_splitter?(s)
     s =~ @value_split_re
   end
 
+  # Helper function for generating single-capture `Regexp` that, when matching a string bound by the given quotes
+  # or brackets, will capture the content that is between the quotes or brackets.
+  #
+  # @api private
```

Comment:
minor: are we using this in Ruby code?

Reply:
I write yarddoc-style docs out of habit; if we find something else useful, I'd be glad to do that instead.